thundra/integrations/base_integration.py

```diff
@@ -0,0 +1,39 @@
+import time
+import traceback
+from thundra.opentracing.tracer import ThundraTracer
+
+
+class BaseIntegrationFactory(object):
+
+    @staticmethod
+    def create_span(wrapped, instance, args, kwargs, operation_name, integration_class):
+        tracer = ThundraTracer.get_instance()
+        response = None
+        exception = None
+        with tracer.start_active_span(operation_name=operation_name, finish_on_close=True) as scope:
+            try:
+                response = wrapped(*args, **kwargs)
+                return response
+            except Exception as operation_exception:
+                exception = operation_exception
+                raise
+            finally:
+                try:
+                    integration_class().inject_span_info(
```

Comment:
in here, we are creating new instance/object by `integration_class`, right?

Reply:
Yes